examples/test/videosphere/hls.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Videosphere HLS</title>
+    <meta name="description" content="Videosphere HLS - A-Frame">
+    <script src="../../../dist/aframe-master.js"></script>
+  </head>
+  <body>
+    <a-scene stats>
+      <a-assets>
+        <video id="city" preload="true" webkit-playsinline playsinline
+               src="../../assets/video/city_halfres_hls.m3u8"
+               width="360" height="360" autoplay loop
+               crossorigin="anonymous"></video>
+      </a-assets>
+
+      <a-entity material="shader: flat; src: #city" geometry="primitive: sphere; radius: 100"
+                scale="1 1 -1">
+      </a-entity>
+      <a-text value="Click or tap to play." align="center" position="0 0.25 -2"></a-text>
+      <script>
+        setTimeout(function() {
+          var vid = document.querySelector('video');
+          if (vid.paused) {
+            window.addEventListener('click', function playOnce() {
+              vid.play();
+              var txt = document.querySelector('a-text');
+              txt.parentElement.removeChild(txt);
+              window.removeEventListener('click', playOnce);
+            });
+          }
+        }, 3000);
```

Comment:
a-assets isn't doing anything, I removed it

Reply:
it doesn’t prevent the loading, but it throws an uncaught Promise or doesn’t resolve (because the `bufferedSeconds` value is wrong - and because of that `if` statement I referenced in other comments).

so, yeah, should be fine - checking for `vid.paused` should be fine without waiting for some event. I didn’t test that too closely. confirmed on your end?

Reply:
seems to work - no prompt on Edge, prompts and plays properly when clicked on macOS Safari.  I can't test on iOS 10.x as mentioned previously